Pcap++/src/PcapFileDevice.cpp

```diff
@@ -560,6 +560,9 @@ void PcapFileWriterDevice::flush()
 
 void PcapFileWriterDevice::close()
 {
+	if (!m_DeviceOpened)
+		return;
+

```

Comment:
In addition to this change, maybe we also want to add a check before calling `fclose(m_File)` to verify that `m_File != NULL`?

Reply:
Done.